Week-1/Session4Players.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Starting lineup</title>
+</head>
+<body>
+    <form>
+        <h1>Starting lineup</h1>
+        <p>
+        <fieldset>
+            <legend>Status :</legend>
+        <input type="radio" id="draft" name="status" value="Draft">
+        <label for="draft">Draft</label>
+        <input type="radio" id="final" name="status" value="Final" checked>
+        <label for="final">Final</label>
+        </fieldset>
+        </p>
+        <p>
+            <label>Date :</label>
+            <select name="gameDate">
+                <option>March 20</option>
+                <option>March 24</option>
+                <option>April 2</option>
+                <option>April 6</option>
+                <option>April 13</option>
+            </select>
+        </p>
+        <p>
+        <fieldset>
+            
+            <legend>Players :</legend>
+            <div>
+            <input type="checkbox" id="p1" name="player" value="Aarav Agarwal">
+            <label for="p1">Aarav Agarwal</label>
+            </div>
+            <div>
+            <input type="checkbox" id="p2" name="player" value="Ava Johnson">
```

Comment:
name should be different for each field, just like id
it helps in referencing that particular input when listening to change event